wai-extra/Network/Wai/Middleware/RequestLogger.hs

```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE DuplicateRecordFields #-}
```

Comment:
I added this because `mFilterRequests` is used in both `DetailedWithSettings` and `ApacheWithSettings`. I noticed that I needed to add the same pragma to my own code using `RequestLogger` - wasn't able to figure out how to import only `ApacheWithSettings`, yet.

So I guess this is a breaking change for everyone importing the whole module and not using this pragma so far.

Any better idea?

Reply:
Don't use the same field names, use something else instead. This is not a change I'd merge.